src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,83 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    @BeforeAll
+    static void setUp() {
+        registrationService = new RegistrationServiceImpl();
```

Comment:
According to the checklist, you need to figure out which way is better to initialize your `RegistrationServiceImpl` class instance - `@BeforeEach` or `@BeforeAll`. It's better to use `@BeforeEach` because it will create a new instance of `RegistrationServiceImpl` for each test case, which will make the tests independent of each other.